src/modules/cmdpal/Microsoft.CmdPal.UI/ExtViews/ListPage.xaml

```diff
@@ -40,13 +40,17 @@
                 BorderThickness="1"
                 CornerRadius="4">
                 <StackPanel Orientation="Horizontal">
```

Comment:
XAML thought, can the parent Border and the StackPanel be merged here? @niels9001?

In general, we want to keep the XAML Tree as flat as possible for performance reasons.

Reply:
(i merged it down)